mod_reforged/hooks/entity/tactical/enemies/bandit_poacher.nut

```diff
@@ -28,17 +28,61 @@
 		this.getSprite("helmet_damage").Saturation = 0.85;
 		this.getSprite("shield_icon").Saturation = 0.85;
 		this.getSprite("shield_icon").setBrightness(0.85);
-		// this.m.Skills.add(this.new("scripts/skills/actives/rotation")); // Replaced with perk
-		// this.m.Skills.add(this.new("scripts/skills/actives/recover_skill")); // Replaced with perk
+		// this.m.Skills.add(::new("scripts/skills/actives/rotation")); // Replaced with perk
```

Comment:
You say rotation is replaced with perk but there is no Rotation perk on this character.